packages/@ember/octane-addon-blueprint/package.json

```diff
@@ -23,5 +23,8 @@
   "devDependencies": {
     "eslint": "^5.12.1",
     "eslint-plugin-node": "^8.0.1"
+  },
+  "resolutions": {
+    "octane-blueprint-utils": "link:../octane-blueprint-utils"
```

Comment:
what happens here in a deployed version of `octane-addon-blueprient` or `octane-app-blueprint`?

Reply:
@NullVoxPopuli the `resolutions` are only used when locally running `yarn` in the package itself and have no effect on installation from npm

Reply:
would it make sense to convert this project to lerna? or do you think that's too heavy?

Reply:
Although, it seems that this is not actually working, I'm not 100% sure why yet though.

I'd like to just land this, bump to 0.1.0 of the utils and call it a day...